corehq/apps/hqwebapp/management/commands/copy_required_static_files.py

```diff
@@ -0,0 +1,133 @@
+import os
+import shutil
+import zipfile
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.apps import apps
+
+
+class Command(BaseCommand):
+    help = ('Copies static files from STATIC_ROOT directory to REQUIRED_STATIC_FILES directory, '
+            'excluding node_modules and only including files from installed apps, js18, webpack, and webpack_b3')
+
+    def handle(self, *args, **options):
+        static_root = settings.STATIC_ROOT
+
+        if not static_root:
+            self.stderr.write("STATIC_ROOT is not configured")
+            return
+
+        REQUIRED_STATIC_FILES_DIR = 'REQUIRED_STATIC_FILES'
+        # Create STATIC_FILTERED directory in project root
+        static_filtered = os.path.join(settings.BASE_DIR, REQUIRED_STATIC_FILES_DIR)
+
+        # Create directory if it doesn't exist, or clean it if it does
+        if os.path.exists(static_filtered):
+            shutil.rmtree(static_filtered)
+        os.makedirs(static_filtered)
+
+        def get_app_name(app_path):
+            """Extract the final app name from a Python path."""
+            # Handle both string paths and config objects
+            if hasattr(app_path, 'name'):
+                app_path = app_path.name
+            # Split on dots and get the last part
+            parts = app_path.split('.')
+
+            # Handle special cases like 'django.contrib.admin' -> 'admin'
+            if 'contrib' in parts:
+                return parts[-1]
+
+            # Handle cases like 'corehq.apps.users' -> 'users'
+            if 'apps' in parts:
+                return parts[-1]
+
+            # Default to last part of path
+            return parts[-1]
```

Comment:
Given that this is the default, the previous two conditions (`'contrib' in parts` and `'apps' in parts`) that also `return parts[-1]` seem redundant. Would it make sense to remove them?

```suggestion
            return app_path.rsplit('.', 1)[-1]
```